spring-cloud-contract-verifier/src/main/java/org/springframework/cloud/contract/verifier/util/MapConverter.java

```diff
@@ -123,8 +126,8 @@ else if (parsed instanceof List) {
 		else if (value instanceof Map) {
 			return convert((Map) value, function, parsingFunction);
 		}
-		else if (value instanceof List) {
-			return ((List) value).stream().map((v) -> transformValues(v, function, parsingFunction))
+		else if (value instanceof List || value instanceof Set) {
```

Comment:
Can we actually make this `value instanceof Collection` ?

Reply:
I thought about it.
If we treat all collections (including stack or queue) as `Map<Key, List<>>` we definitely can. But should we? 
Json has arrays and arrays has an order, that's why I'm not sure that it's a good idea 
to treat `Map<Key,Set<>>` as `Map<Key,List<>>`. 
On the other hand, from the "business logic" POV we know that nobody would use `Map<Key, Queue<>>` or some other collections.

So, @marcingrzejszczak  It's up to you. I updated, but can revert if you wish :)